.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,32 +30,31 @@ N/A
-- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
-- [ ] For dependency updates: links to external changelogs and, if possible, full diffs
-
-<!-- For new API and extension points: Link to the reference implementation in open-source (or example in Javadoc) -->
+- [ ] The Jira issue, if it exists, is well-described.
+- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
+  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
+- [ ] There is automated testing or an explanation as to why this change has no tests.
+- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
+- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
+- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
+- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
+- [ ] For new APIs and extension points, there is a link to at least one consumer.
 
 ### Desired reviewers
 
 @mention
 
 <!-- Comment:
-If you need an accelerated review process by the community (e.g., for critical bugs), mention @jenkinsci/code-reviewers
+If you need an accelerated review process by the community (e.g., for critical bugs), mention @jenkinsci/core-pr-reviewers.
 -->
 
 ### Maintainer checklist
 
 Before the changes are marked as `ready-for-merge`:
 
-- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
-- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
-- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
-- [ ] Proper changelog labels are set so that the changelog can be generated automatically
-- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
+- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
```

Comment:
could you explain the reason you added two and (2)?
```suggestion
- [ ] There are at least two approvals for the pull request and no outstanding requests for change.
```

Reply:
https://www.quora.com/Why-do-people-sometimes-write-numbers-like-two-2-or-fourteen-14

Reply:
The top answer says:
> It seems a redundant and unnecessary practice.

Reply:
Only in the opinion of some. This is common in formal technical writing, just like putting the section of the manual page in parentheses.